RatScanner/ViewModel/SettingsVM.cs

```diff
@@ -91,6 +104,23 @@ public void LoadSettings() {
 		BlurBehindSearch = RatConfig.Overlay.Search.BlurBehind;
 		InteractableOverlayHotkey = RatConfig.Overlay.Search.Hotkey;
 
+		// Floating tooltip
+		IsFloatingTooltipEnabled = RatConfig.FloatingTooltip.Enable;
+		IsFloatingTooltipShowName = RatConfig.FloatingTooltip.ShowName;
+		IsFloatingTooltipShowAvgDayPrice = RatConfig.FloatingTooltip.ShowAvgDayPrice;
+		IsFloatingTooltipShowPricePerSlot = RatConfig.FloatingTooltip.ShowPricePerSlot;
+		IsFloatingTooltipShowTraderPrice = RatConfig.FloatingTooltip.ShowTraderPrice;
+		IsFloatingTooltipShowUpdated = RatConfig.FloatingTooltip.ShowUpdated;
+		IsFloatingTooltipShowTasksInfo = RatConfig.FloatingTooltip.ShowTasksInfo;
+		IsFloatingTooltipShowHideoutInfo = RatConfig.FloatingTooltip.ShowHideoutInfo;
+		IsFloatingTooltipShowBarterInfo = RatConfig.FloatingTooltip.ShowBarterInfo;
+		IsFloatingTooltipShowCraftsInfo = RatConfig.FloatingTooltip.ShowCraftsInfo;
+
+		if (IsFloatingTooltipEnabled)
+		{
+			FloatingTooltip.Instance.Hide();

```

Comment:
To instantiate the FloatingTooltip Instance when needed and to skip it if it is disabled.